doc/man3/EVP_MD_fetch.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-EVP_MD_fetch, EVP_CIPHER_fetch
+EVP_MD_fetch, EVP_CIPHER_fetch, EVP_EXCHANGE_fetch
```

Comment:
The pod file should be renamed now.

Reply:
Renamed to what? All our pod files normally take the name of one of the functions defined within it. All the other functions have sym links created for them to point at the pod file.

Reply:
Ahh ok - I wasnt aware it worked like that. Ideally the pod file would be EVP_fetch.pod